### DISCUSSION TITLE
How to backup my setting[other]

### DISCUSSION THREAD

**Discussion by rinldo** [noah-nuebling/mac-mouse-fix#638] [Questions] [answered]
Hi there,
Thank you for developing this app for mac users.
May I ask how to backup my settings on this app?
Thanks

**Comment by noah-nuebling** [comment, marked as answer]
Hi! There is no official way to back up your settings, but here's a hacky way how you could do it:

 If you go to the "More Info..." screen (or the "About" screen in MMF 3), and then click the Mac Mouse Fix icon while holding Command and Shift on your keyboard, a file called "config.plist" file will be revealed in the Finder. This file contains all your settings. You could store it somewhere else to back it up. Then, later you could restore the backup by replacing the "config.plist" file with your backed up version. (Make sure Mac Mouse Fix is closed while you replace the file)

Hope this helps and let me know if you have more questions!

**Reply by vii33** [reply]
@noah-nuebling Thank you for the reply. I would be great if the file location could be mentioned in the github readme so people can find it easier. Also a "standard link" in the "About" section (instead of a hidden keybinding) would be great. I would guess this is helpful for many people, no need to hide it.
